rom/laghos_rom.hpp

```diff
@@ -383,6 +384,7 @@ class ROM_Sampler
                 }
             }
         }
+        else first_sv = 1;
```

Comment:
Isn't this the case of no offsets? Does the first SV still need to be skipped in that case?

Reply:
Yes this is the case of no offsets, which is rarely used. However, if it is used, it has the same issue: there is one single principal component which eats up too much energy factor (even in shorter time simulation in this case). So I do the same as in the initial offset.